vtr_flow/tasks/mixing_optimization/correctness_custom/mults_and_adders_simple/config/multconfig.xml

```diff
@@ -0,0 +1,29 @@
+	<config>
```

Comment:
idem

Reply:
THat's meant to be used for flow though.... Would you suggest introducing a folder in Odin subdirectory that will actually run tasks?

Reply:
yeah! just create a task for it, and if you have many different config, just make a directory with multiple task, and create a suite to drive it